asciidoc/volume2/gateways/tf2-ch-b-gateway-private-mdc-mapping.adoc

```diff
@@ -8,6 +8,18 @@ Coded elements in SDC using *MDC* private codes require also to contain a *pm:Tr
 
 Please refer to <<ref_ieee_11073_10700_2022>> *TR1358* for further information.
 
+.R8119
+[sdpi_requirement#r8119,sdpi_req_level=shall,sdpi_max_occurrence=2]
+****
+A <<actor_somds_provider>> shall provide exactly one *pm:Translation* where *pm:Translation/@Code* is identical with *pm:CodedValue/@Code*.

```

Comment:
Proposed extension/clarification:
"For each private code a SOMDS Provider shall [...]"